src/components/styles/NavBar.styled.ts

```diff
@@ -1,13 +1,13 @@
 import styled from 'styled-components'
 
 export const MainContainer = styled.div`
-  width: 65%;
-  display: flex;
-  margin-left: 10%;
-  justify-content: space-between;
+  width: 100vw;
+  ${(props) => `background-color: ${props.theme.palette.common.black};`}
 `
 
 export const NavContainer = styled.div`
-  width: 100vw;
-  background: ${(props) => props.theme.palette.common.black};
+  width: 65%;
```

Comment:
Do the Nav Items overlap or display fully on mobile?

Reply:
Fixed in commit 2118858